apps/comps/hooks/useAuth.ts

```diff
@@ -36,15 +38,23 @@ export const useNonce = () => {
 export const useLogin = () => {
   const queryClient = useQueryClient();
   const [, setUserAtom] = useAtom(userAtom);
+  const { trackEvent } = useAnalytics();
+  const posthog = usePostHog();
 
   return useMutation({
     mutationFn: async (data: LoginRequest) => {
       const result = await apiClient.login(data);
       return result;
     },
-    onSuccess: () => {
+    onSuccess: (response, _) => {
```

Comment:
the `, _` is unnecessary.

note: i'm not sure why our `pnpm lint` command doesn't pick this up. i'll look into that separately.